Specs/DateTime/German/DateTimeModel.json

```diff
@@ -446,50 +476,52 @@
             {
               "timex": "2016-11-07TEV",
               "type": "datetimerange",
-              "start": "2016-11-07 16:00:00",
-              "end": "2016-11-07 20:00:00"
+              "start": "2016-11-07 18:00:00",
+              "end": "2016-11-07 22:00:00"
             }
           ]
         }
       }
     ]
   },
   {
-    "Input": "Das ist dann nächsten Montag Nachmittag.",
+    "Input": "Das ist dann nächsten Montagnachmittag.",
```

Comment:
Is the two-token form not commonly used?

Reply:
It commonly used, but incorrect. But as many people will use the incorrect form I now added this back in, but also left the correct form in there.